bpf/kmesh/cgroup_sock.c

```diff
@@ -30,6 +30,16 @@
 #if KMESH_ENABLE_IPV4
 #if KMESH_ENABLE_HTTP
 
+/*
+ * This map is used to store the cookie information
+ * of pods managed by kmesh. The key represents the
+ * cookie, and the value represents whether it is bypassed.
+ * The default value is 0, indicating it is not
+ * bypassed by default. A value of 1 represents bypassed
+ * status. Whether it is managed by kmesh is unrelated
+ * to the value. The only determining factor is whether
+ * there is cookie information for this pod in the map.
+ */
```

Comment:
Pls add design description to PR description and update the test report

Reply:
> Pls add design description to PR description and update the test report

done